docs/blog/2019-02-20-introducing-use-static-query/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Introducing useStaticQuery
+date: 2019-02-20
+author: Sidhartha Chatterjee
+tags:
+  - hooks
+  - static query
+---
+
+React Hooks are _cool_. Besides simplifying your code and removing the need for a lot of boilerplate associated with classes in JavaScript (looking at you, `this`), they enable some serious shareability. They also make it possible to use state in functional components.
+
+You can probably tell that we've been super excited about [React Hooks][hooks-intro]. So when they finally landed in React 16.8, we figured it was time to give our very own `StaticQuery` component the hook treatment!
+
+### Say hello to useStaticQuery
+
+`useStaticQuery` is a hook that takes a GraphQL query and returns your data. That's it.
+
+**No more Render Props necessary to use a Static Query**
+
+This simplifies accessing data in your components and also keeps your component tree shallow.
+
+Let's quickly check out a simple example. Here's a simple Header component, first using `StaticQuery` and then with `useStaticQuery`.
```

Comment:
If possible, it would be good to change usage of the word "simple" to something less assuming. How about "check out an example", and  "Here's a basic Header component"?

Reply:
This is a great point! 

> Let's quickly check out a basic example. Here's a typical Header component,

What do you think of this?

Reply:
That works! :shipit: